polaris-react/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -77,6 +77,24 @@ export function Error() {
   );
 }
 
+export function Magic() {
+  const [checked, setChecked] = useState<CheckboxState>(false);
+  const handleChange = () => {
+    console.error('This should never be fired');
+  };
+
+  return (
+    <InlineStack gap="200">
+      <Checkbox
+        label="Basic checkbox"
+        checked={checked}
+        onChange={handleChange}
```

Comment:
polaris-react needs to be built for the new prop to work in stories

```suggestion
        onChange={handleChange}
        tone="magic"
```

Reply:
This was missing from my side 😞